vissl/utils/misc.py

```diff
@@ -80,6 +80,19 @@ def is_apex_available():
     return apex_available
 
 
+def is_augly_available():
```

Comment:
could we also assert somewhere about the python version if augly has a specific requirement?